Sming/Core/Network/Http/BasicHttpHeaders.h

```diff
@@ -138,14 +138,6 @@ class BasicHttpHeaders : public HttpHeaderFields
 		parser.method = unsigned(method);
 	}
 
-	/**
-	 * @brief Obtain text for request method
-	 */
-	const char* methodStr() const
```

Comment:
> Could do a similar thing for http_status, however it's going to break a lot of applications (more so than HttpMethod)

Nooo... it's too much ... The commit `Re-define HttpMethod as strong enum` is a good idea but I would like to delay this for a later release as it will break already all Http related apps. Can you do a minimal thing: rename `methodStr()` to `methodAsString()`. Or `getMethodText()` ?

Reply:
I have a solution in mind which won't break things... bear with.

Reply:
@slaff OK, have added HttpMethod, HttpStatus and HttpError seems OK... Whadya think?

Reply:
>  Whadya think?

Thanks, looks much better now. I will look at it closer later today.